crates/pink/pink-extension/Cargo.toml

```diff
@@ -29,4 +29,3 @@ std = [
 ]
 runtime_utils = ["std"]
 dlmalloc = ["ink/no-allocator", "dep:dlmalloc"]
-ink-as-dependency = []
```

Comment:
This feature flag is unused but breaks the drink to work. Let's remove it.
Because of this breaking change, we bump the version to `0.5.0`.